tests/extensions/test_timestamps.py

```diff
@@ -0,0 +1,42 @@
+import json
+import unittest
+from datetime import datetime
+
+from pystac import Item, STACError
+from pystac.utils import str_to_datetime
+from tests.utils import TestCases, test_to_from_dict
+
+
+class TimestampsTest(unittest.TestCase):
+    LANDSAT_EXAMPLE_URI = TestCases.get_path('data-files/timestamps/example-landsat8.json')
+
+    def setUp(self):
+        self.maxDiff = None
+
+    def test_to_from_dict(self):
```

Comment:
Consider using the test layout that's used in other extension test, e.g. [test_view.py](https://github.com/azavea/pystac/blob/develop/tests/extensions/test_view.py), which makes a single test for each property and clearly tests the get and set. Ideally the layout for tests is consistent for each extension.

I'm also not sure how much value the test_to_from_dict is doing here - that is testing Item functionality which isn't the subject of these tests.

Reply:
Good call. I rewrote all the tests sticking to the `test_view.py` format. I left the `test_to_from_dict` in for consistency but can take it out if you want